cmd/minikube/cmd/service.go

```diff
@@ -187,6 +187,11 @@ func startKicServiceTunnel(services service.URLs, configName, driverName string)
 		sshPort := strconv.Itoa(port)
 		sshKey := filepath.Join(localpath.MiniPath(), "machines", configName, "id_rsa")
 
+		// check whether all pods of this service is crashed
```

Comment:
Hey, I think @spowelljr suggested some changes. Are you still working on this? Thanks!

Reply:
Yes, I am still working on this, but which changes? I thought all changes suggested by @spowelljr has been accepted, and is there anything else I should do? Did I miss anything?

Reply:
Sorry I didn't understand, could you please clearly tell me what changes should I do next? I did't see anything changes requested are not accepted